src/components/toDoInput.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// CSS
+const input_css = {
+  fontFamily: 'Open Sans, Helvetica Neue, Helvetica, Arial, sans-serif',
+  fontSize: '14px',
+  lineHeight: '2em',
+  width: '94%',
+  border: 'none',
+  borderBottom: '1px solid #ddd',
+  backgroundColor: ' #f5f5f5',
+  marginLeft: 10
+};
+
+// Class Component
+class ToDoInput extends React.Component {
+  constructor(props) {
+    super(props);
+    // this.state.value is <input> value
+    this.state = { value: '' };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  }
+  handleSubmit(event) {
```

Comment:
Instead of binding this to a keyDown event, you might find it easier to bind it to an onSubmit event. That way you don't have to do any logic to guess the key.